ngraph/core/reference/include/ngraph/runtime/reference/ctc_loss.hpp

```diff
@@ -118,85 +118,86 @@ namespace ngraph
                     T res = -type_inf;
 
                     // Looking for aligned paths
-                    std::function<void(size_t, size_t, size_t, T)> findPaths = [&](
-                        size_t targetIdx, size_t start, size_t end, T prevLogProb) {
-                        if (end > actualLogitLen)
-                        {
-                            if (res == -type_inf)
-                            {
-                                res = prevLogProb;
-                            }
-                            else if (prevLogProb != -type_inf)
+                    std::function<void(size_t, size_t, size_t, T)> findPaths =
+                        [&](size_t targetIdx, size_t start, size_t end, T prevLogProb) {
+                            if (end > actualLogitLen)
                             {
-                                if (res > prevLogProb)
-                                    res = res + std::log1pf(std::exp(prevLogProb - res));
-                                else
-                                    res = prevLogProb + std::log1pf(std::exp(res - prevLogProb));
+                                if (res == -type_inf)
+                                {
+                                    res = prevLogProb;
+                                }
+                                else if (prevLogProb != -type_inf)
+                                {
+                                    if (res > prevLogProb)
+                                        res = res + std::log1pf(std::exp(prevLogProb - res));
+                                    else
+                                        res =
+                                            prevLogProb + std::log1pf(std::exp(res - prevLogProb));
```

Comment:
Has the maximum line length changed?

Reply:
I didn't modify old configuration rules.
https://github.com/openvinotoolkit/openvino/blob/a20c31f70065f12695eddd00312cc25ba0f3523b/ngraph/.clang-format#L26